### ISSUE TITLE
Promote aarch64-apple-ios-sim to Tier 2

### ISSUE THREAD

**Issue by badboy** [rust-lang/rust#82412]
With #81966 landed it's possible to build for the arm64 iOS simulator on M1 devices.

```
cargo +nightly build -Z build-std --target aarch64-apple-ios-sim
```

It would be nice to promote `aarch64-apple-ios-sim` to Tier 2 so users can get the pre-built `libstd` using rustup.

What steps are necessary to do that? I'm happy to do the work.
And who's to sign off on that?

/cc @deg4uss3r who did the initial work.

**Comment by jyn514** [comment]
You may want to ask [T-infra](https://rust-lang.zulipchat.com/#narrow/stream/242791-t-infra) on Zulip, you'll likely get more feedback there.

**Comment by badboy** [comment]
> You may want to ask [T-infra](https://rust-lang.zulipchat.com/#narrow/stream/242791-t-infra) on Zulip, you'll likely get more feedback there.

Done.

**Comment by daltonclaybrook** [comment]
Any update on this?

**Comment by badboy** [comment]
With the [Target tier policy](https://doc.rust-lang.org/nightly/rustc/target-tier-policy.html) written out I plan to post a MCP next week to get the process kicked off.

**Comment by badboy** [comment]
[MCP posted](https://github.com/rust-lang/compiler-team/issues/428)

**Comment by badboy** [comment]
Closed by #87760
